src/stock-adjustment-creation/adjustment-creation.service.js

```diff
@@ -29,11 +29,11 @@
         .service('stockAdjustmentCreationService', service);
 
     service.$inject = [
-        '$filter', 'StockEventRepository', 'openlmisDateFilter',
+        '$filter', 'StockEventRepository', 'openlmisDateFilter', 'localStorageService',
         'messageService', 'productNameFilter', 'dateUtils', '$rootScope'
     ];
 
-    function service($filter, StockEventRepository, openlmisDateFilter,
+    function service($filter, StockEventRepository, openlmisDateFilter, localStorageService,
```

Comment:
Please add comments that there is customization (like it has been made in lines 83 and 85)